ref(loader): Add analytics when loader is rendered

mydea opened the pull request:
This adds analytics for when the loader is rendered, so we can get better data in that area.

AbhiPrasad commented on src/sentry/web/frontend/js_sdk_loader.py:
```diff
@@ -171,6 +188,17 @@ def get(self, request: Request, public_key: Optional[str], minified: Optional[st
+            has_replay=loader_config["hasReplay"],
+            has_debug=loader_config["hasDebug"],
+            sdk_version=sdk_version,
+            tmpl_instance=instance,
```
m: Can we just store `tmpl` instead (`sentry/js-sdk-loader-noop.js.tmpl`)? Feel like that is easier to understand than default/noop/minified.

mydea replied:
sure thing!

AbhiPrasad reviewed (commented):
Perhaps @scefali can help us decide if this is ok amplitude cost wise.

AbhiPrasad reviewed (approved):
LGTM! Might be interesting to watch the sentry dashboard (https://sentry.sentry.io/dashboard/38857/?project=1&statsPeriod=24h) and see if load times are affected by recording these metrics. I assume not, but maybe we discover something!

mydea commented:
FYI after looking at the Big Query table & other example analytic events, we updated the naming to `organization_id` & also added `project_id` because these are handled specially.

priscilawebdev commented:
@mydea this is when the loader is rendered in the UI? why did we decide to go with analytics on the backend?

mydea commented:
> @mydea this is when the loader is rendered in the UI? why did we decide to go with analytics on the backend?

Because it is the easiest way to capture this correctly, basically :)
